general_considerations.md

```diff
@@ -194,6 +262,13 @@ bitcoin. The "UTXO set" of the consensus mechanism is the set of payment outputs
 for all hashers, with amounts decided by the recorded shares and consensus
 mechanism rules.
 
+[kp]
+
+What are the inputs to these payment outputs for hashers?  They can't
+all have the same coinbase input, so what are they spending?
+
+[/kp]
+
```

Comment:
The inputs are all past coinbase outputs "owned" by the pool. These can be rolled into one output by combining the previous coinbase output with each new block's coinbase, so there's only one coinbase output with each block (and its value is the sum total of funds controlled by the pool).